<launch>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find my_turtlebot3_openai_example)/config/my_turtlebot3_openai_ppo2_params.yaml" />
    <!-- Launch the training system -->
    <node pkg="my_turtlebot3_openai_example" name="turtlebot3_world" type="train_turtlebot3_PPO2.py" output="screen"/>
</launch>
